setup.py

```diff
@@ -70,8 +70,19 @@ def _find_version(*file_paths):
         "emcee<3",
         "powerbox>=0.5.7",
         "cached_property",
-        "21cmFAST>=3.0.0dev1",
+        "21cmFAST>=3.0.0dev6",
```

Comment:
Should update this to be `21cmFAST>=3.0<4.0`

Reply:
okay!